librz/bin/format/java/class_bin.c

```diff
@@ -1040,7 +1040,7 @@ RZ_API RZ_OWN RzList *rz_bin_java_class_strings(RZ_NONNULL RzBinJavaClass *bin)
 }
 
 static char *add_class_name_to_name(char *name, char *classname) {
-	if (classname && name) {
+	if (classname) {
```

Comment:
This is not a bug. name cannot be NULL, but coverity says that is a NULL deref because of an old check which is always NONNULL